.github/workflows/swift.yml

```diff
@@ -26,9 +26,7 @@ jobs:
       run: xcodebuild -version
     - name: Check Swift version
       run: swift --version
-    - name: Build
-      run: swift build --build-tests
```

Comment:
wouldn't keeping the two steps and using `swift build --build-tests --enable-test-discovery` make it more obvious where the build failed?

Reply:
Well yes, that's the reason why it was that way. But that's the sacrifice we have to make in oder to not double compile. I haven't found a `swift build` argument were it compiles in codecov instructions.

Reply:
I mixed up --enable-code-coverage and  --enable-test-discovery 🙈

Reply:
Was a bit confused of your comment but now I understand where it came from 😂👍